src/components/Dashboard/ThisWeek/NewsletterMenu/NewsletterMenu.test.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { GqlMockedProvider } from '../../../../../__tests__/util/graphqlMocking';
+import {
+  GetEmailNewsletterContactsQuery,
+  GetTaskAnalyticsQuery,
+} from '../NewsletterMenu.generated';
+import NewsletterMenu from './NewsletterMenu';
+
+const accountListId = '111';
+
+const mockEnqueue = jest.fn();
+
+describe('NewsletterMenu', () => {
+  it('default', async () => {
+    const { queryByText } = render(
+      <GqlMockedProvider<
+        GetTaskAnalyticsQuery & GetEmailNewsletterContactsQuery
+      >>
+        <NewsletterMenu accountListId={accountListId} />
+      </GqlMockedProvider>,
+    );
+
+    expect(queryByText('Never')).toBeNull();
+    expect(queryByText('Newsletter')).toBeVisible();
+  });
+
+  it('should open newsletter menu', async () => {
+    const { queryByText, queryByTestId } = render(
+      <GqlMockedProvider<
+        GetTaskAnalyticsQuery & GetEmailNewsletterContactsQuery
+      >>
+        <NewsletterMenu accountListId={accountListId} />
+      </GqlMockedProvider>,
+    );
+
+    userEvent.click(queryByTestId('NewsletterMenuButton'));
+
+    expect(queryByText('Log Newsletter')).toBeInTheDocument();
+    expect(queryByText('Export Email')).toBeInTheDocument();
+    expect(queryByText('Export Physical')).toBeInTheDocument();
+  });
+
+  describe('Log Newsletter', () => {
```

Comment:
As I mentioned, I created super barebones tests for the two menu options that I didn't fully fledge out in this pr.